lib/css/CssParser.js

```diff
@@ -560,6 +576,9 @@ class CssParser extends Parser {
 							modeData = "local";
 							const dep = new ConstDependency("", [start, end]);
 							module.addPresentationalDependency(dep);
+						} else if (pseudoFunctionNames.includes(name)) {
+							awaitRightParenthesis = true;
+							modeStack.push(false);
```

Comment:
Sorry for delay, my bad, let's remove it and move `awaitRightParenthesis = true;` to `else`, so we will not to add more pseudo in future and will always generate `)`

Reply:
done